### DISCUSSION TITLE
Reminder to review habits of the day

### DISCUSSION THREAD

**Discussion by navinthomas** [iSoron/uhabits#1400] [Feature Requests]
It would be nice to have a review notification just to remind us to review the habits for a day, so that I would then go over the habit list and mark those that I have done as done, and those that I have not done as skipped or not done.

If there are many habits that need to be done in a day, setting reminders for each of them would clutter the notification space with many reminder notifications. If there is a review notification, then I can keep reminders for only the critical habits. For the rest, I can rely on the review notification to remind me to mark them as done or not done.

The review notification can just be a simple notification that can be scheduled for a specific time every day, like 9 PM or 10 PM, so that I am reminded to review the habits before going to bed. Or it can be a more sophisticated notification that shows up at the scheduled time only if there are unmarked habits for that day. Or it just shows a message like "5 out of 7 habits for today have been done". There are different ideas possible for the notification, but ultimately, its main purpose is to make the user go over that day's habit list and updates the statuses of those habits that were not marked.

**Comment by hiqua** [comment]
What about creating a new habit named "review habits" and setting a corresponding reminder?

**Reply by navinthomas** [reply]
I did think about that, but it just felt like a workaround or an incomplete solution. Not to mention the fact that it adds another unnecessary entry in the habit list.

**Reply by hiqua** [reply]
Yes of course. I think it's not that bad a workaround though. Another one is to just set an alarm everyday at the desired time.

Can you make a list of features you'd like to have in your list review notification which would make it more meaningful to implement directly in the app? I can see:
* customize the time
* appear depending on how many habits are uncompleted and show this number

What else?

**Reply by navinthomas** [reply]
Apologies for not responding sooner.

Honestly, I can't think of any additional features for the notification at the moment. Customising the time, and showing a summary of the uncompleted habits of the day would be enough IMO.

**Comment by toupeira** [comment]
I would like this too, I currently use a custom Tasker profile to show a daily notification, with an action button to open Loop.

Having a summary of uncompleted habits in the notification would be very nice.
